samples/cellular/nrf_cloud_multi_service/overlay_coap.conf

```diff
@@ -19,10 +19,13 @@ CONFIG_COAP_FOTA_JOB_CHECK_RATE_MINUTES=5
 CONFIG_LOG=y
 CONFIG_NET_LOG=y
 CONFIG_LOG_PRINTK=y
-CONFIG_COAP_LOG_LEVEL_INF=y
 CONFIG_LOCATION_LOG_LEVEL_INF=y
-CONFIG_NRF_CLOUD_COAP_LOG_LEVEL_INF=y
-CONFIG_MULTI_SERVICE_LOG_LEVEL_INF=y
+CONFIG_COAP_LOG_LEVEL_DBG=y
+CONFIG_NRF_CLOUD_COAP_LOG_LEVEL_DBG=y
+CONFIG_MULTI_SERVICE_LOG_LEVEL_DBG=y
+CONFIG_LTE_LINK_CONTROL_LOG_LEVEL_INF=y
+CONFIG_NET_CONNECTION_MANAGER_LOG_LEVEL_DBG=y
+CONFIG_NRF_MODEM_LIB_NET_IF_LOG_LEVEL_DBG=y
```

Comment:
The PR is no longer in draft mode, and the commit mentions you were planning to clean this up when you exited that, is that still the case?

Reply:
I fixed that earlier but had not yet pushed. It's updated now.